doc/nrf/ug_thread_tools.rst

```diff
@@ -115,3 +142,120 @@ The most common wpanctl commands are the following:
   For example, ``set NCP:SleepyPollInterval 1000`` will set the value of the ``NCP:SleepyPollInterval`` property to ``1000``.
 
 For the full list of commands, run the ``help`` command in wpanctl.
+
+.. _ug_thread_tools_pyspinel:
+
+PySpinel
+********
+
+`PySpinel`_ is a tool for controlling OpenThread Network Co-Processor instances through a command-line interface.
+
```

Comment:
Is there anything else we should state here? Is the tool available for Windows, or should we have a note like we have for wpantund?

Reply:
It the same like for wpantund, Linux and Mac OS only.

Reply:
Added.